xc/device.ml

```diff
@@ -1779,6 +1784,12 @@ module Dm = struct
       if not !finished then
         raise (Ioemu_failed (name, "Timeout reached while starting daemon"))
     end;
+
```

Comment:
Can we put this in a function wakeup() inside the module QMP_Event and call this function here?

Reply:
Done, thanks